extensions/resource-deployment/src/ui/deployAzureSQLDBWizard/pages/databaseSettingsPage.ts

```diff
@@ -128,20 +150,87 @@ export class DatabaseSettingsPage extends BasePage {
 		this._endIpAddressTextRow = this.wizard.createFormRowComponent(view, constants.EndIpAddressLabel, '', this._endIpAddressTextbox, true);
 	}
 
-	private createFirewallNameText(view: azdata.ModelView) {
+	/**
+	 * Firewall rule names shown below are based on the Name field (under "Policy Details") when entering erroneous input here: https://ms.portal.azure.com/#create/Microsoft.FirewallPolicy
+	*/
+	private validateFirewallNameText(firewallname: string | undefined): boolean {
+		if (firewallname) {
+			// Check for firewall rule name that is only spaces (not allowed).
```

Comment:
I assume any whitespace characters are not allowed so you should use /s instead of just space